src/CoverageTransformer.js

```diff
@@ -123,7 +123,7 @@ export default class CoverageTransformer {
 		if (sourceMap.sourcesContent) {
 			origSourceFilename = rawSourceMap.sources[0];
 
-			if (origSourceFilename && path.extname(origSourceFilename) !== '') {
+			if (origSourceFilename && path.extname(origSourceFilename) !== '' && rawSourceMap.sources.length === 1) {
 				origFileName = rawSourceMap.file;
```

Comment:
I did initially try that but it didn't fix the [original issue](https://github.com/marcules/karma-remap-istanbul/issues/31). This was the only solution I could find that worked.

The idea behind this hackish fix is that currently if there are > 1 sources this block will always throw that error and make remap-istanbul unusable. The ideal solution would be to make a proper test case, and fix the underlying issue, however I don't have the knowledge myself to be able to do that 😞